### PR TITLE
Add Bandit to pkd_dev_tasks

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
added [Bandit](https://bandit.readthedocs.io/en/latest/) to run after the linters.
it does not fail the build in any case, but it prints the security issues found, as seen below in the screenshot

## Screenshots
image

### CODE REVIEW

**Comment by glicht** [comment]
I like bandit. Would like to find the right level to enforce on. I think we can do level high confidence high as a start.

**Comment by Itay4** [comment]
@glicht done

image

**Comment by glicht** [comment]
We need to run with the proper python interpreter. Probably simplest is to give up on the run_bandit.sh file and simply call directly the proper python version  as we do with flake8. 

See: https://github.com/PyCQA/bandit#under-which-version-of-python-should-i-install-bandit

File: Tests/scripts/pkg_dev_test_tasks.py [hunk 1]
```diff
@@ -260,6 +261,16 @@ def run_mypy(project_dir, py_num):
     print("mypy completed")
 
 
+def run_bandit(project_dir, py_num):
+    lint_files = get_lint_files(project_dir)
+    print("========= Running bandit on: {} ===============".format(lint_files))
+    python_exe = 'python2' if py_num < 3 else 'python3'
+    print_v('Using: {} to run flake8'.format(python_exe))
```
Comment (glicht) [inline Tests/scripts/pkg_dev_test_tasks.py]:
```suggestion
    print_v('Using: {} to run bandit'.format(python_exe))
```

File: Tests/scripts/pkg_dev_test_tasks.py [hunk 2]
```diff
@@ -260,6 +261,16 @@ def run_mypy(project_dir, py_num):
...
+    sys.stdout.flush()
+    subprocess.check_call([python_exe, '-m', 'bandit', '-lll', '-iii', lint_files], cwd=CONTENT_DIR)
```
Comment (glicht) [inline Tests/scripts/pkg_dev_test_tasks.py]: Let's add `-q`. It prints out a lot of stuff when there is no need.
```suggestion
    subprocess.check_call([python_exe, '-m', 'bandit', '-lll', '-iii', '-q', lint_files], cwd=CONTENT_DIR)
```

**Pull request merged**